fuzz/fuzz_targets/parse.rs

```diff
@@ -0,0 +1,7 @@
+#![no_main]
+#[macro_use] extern crate libfuzzer_sys;
+extern crate goblin;
+
+fuzz_target!(|data: &[u8]| {
+    let _ = goblin::parse(data);
```

Comment:
Have you tried this? Last time iirc, it wouldn't work until I forced it to not pass 0 length bytes.